receive-edi-as2/README.md

```diff
@@ -0,0 +1,111 @@
+# Expose B2B Cloud for AS2 partners to exchange EDI messages
+
+B2B Cloud can be exposed as an AS2 over HTTP channel and used to exchange EDI messages with Partners. This usecase is common for all Partners who has capability of exchanging EDI messages over AS2 which is a very popular protocol for EDI.
+
+Contributors: Chaitanya Jadcherla, Mangat Rai , Shashank Patel
+
+
+## Prerequisites
+1. You need Software AG webmethods.io B2B cloud tenant and webmethods.io integration cloud tenant. If you don't have one; sign up for free 30 trial tenant at [Software AG B2B](https://signup.softwareag.cloud/#/?product=b2b)
+
+![](https://github.com/patelshashank/webmethods-b2b-examples/blob/master/receive-edi-as2/images/B2BLandingPage.png)
```

Comment:
Change to relative links, for example 
```
~[](images/B2BLandingPage.png)
```

Reply:
Hey Peter. 
I have changed it to relative links. 

Thanks,
Shashank